src/containers/profile/src/caseparticipation/AddPeopleToCaseForm.js

```diff
@@ -0,0 +1,229 @@
+  const caseEKID :?UUID = getEntityKeyId(personCase);
+  const caseNumber = getPropertyValue(personCase, [NOTES, 0]);
+  const caseIdentifier = `Case #: ${caseNumber}`;
+
+  const caseRoleMap :Map = useSelector((store) => store
+    .getIn([PROFILE, PERSON_CASE_NEIGHBOR_MAP, ROLE, caseEKID], Map()));
+  const respondentPerson = caseRoleMap.getIn([RESPONDENT, 0], Map());
+  const respondentPersonName = getPersonName(respondentPerson);
+  const respondentText = `Respondent: ${respondentPersonName}`;
+  const peacemakerPerson = caseRoleMap.getIn([PEACEMAKER, 0], Map());
+  const peacemakerPersonName = getPersonName(peacemakerPerson);
+  const peacemakerText = `Peacemaker: ${peacemakerPersonName}`;
+  const victims = caseRoleMap.get(VICTIM, List());
+  const victimNames = victims.map((victim) => getPersonName(victim)).toJS();
+  const victimText = `Victims: ${victimNames.join(', ')}`;
+
+  const [formInputs, setFormInputs] = useState({ firstName: '', lastName: '' });
+  const [dob, setDOB] = useState('');
+  const [page, setPage] = useState(1);
+  const [modalIsVisible, setModalVisibility] = useState(false);
+  const [selectedPerson, selectPersonForModal] = useState(Map());
+
+  const onInputChange = (e :SyntheticEvent<HTMLInputElement>) => {
+    const { name, value } = e.currentTarget;
+    setFormInputs({ ...formInputs, [name]: value });
+  };
+
+  const dispatch = useDispatch();
+
+  const searchPeopleForCase = (e :SyntheticEvent<HTMLInputElement> | void, startIndex :?number) => {
+    const start = startIndex || 0;
+    dispatch(searchPeople({
+      dob,
+      firstName: formInputs.firstName,
+      lastName: formInputs.lastName,
+      maxHits: MAX_HITS,
+      start,
+    }));
+  };
+
+  const onPageChange = ({ page: newPage, start } :Object) => {
+    searchPeopleForCase(undefined, start);
+    setPage(newPage);
+  };
+
+  const person :Map = useSelector(selectPerson());
+  const personName :string = getPersonName(person);
+  const root :string = useSelector((store) => store.getIn(APP_PATHS.ROOT));
+
+  const searchedPeople :List = useSelector((store) => store.getIn([PROFILE, SEARCHED_PEOPLE]));
+  const totalHits :List = useSelector((store) => store.getIn([PROFILE, TOTAL_HITS]));
+  const data = searchedPeople.map((searchedPerson :Map) => {
+    const name = getPersonName(searchedPerson);
+    const personDOB = getPropertyValue(searchedPerson, [DOB, 0]);
+    return Map({ name, person: searchedPerson, personDOB });
+  });
+
+  const searchPeopleRequestState = useSelector((store) => store.getIn([PROFILE, SEARCH_PEOPLE, REQUEST_STATE]));
+  const isSearching :boolean = isPending(searchPeopleRequestState);
+  const hasSearched :boolean = isFailure(searchPeopleRequestState) || isSuccess(searchPeopleRequestState);
+
+  const resetSearchedPeopleList = () => {
+    dispatch(clearSearchedPeople());
+  };
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  useEffect(() => resetSearchedPeopleList, []);
```

Comment:
`resetSearchedPeopleList` is not used anywhere else. Please move it inside the `useEffect` to avoid creating the function on every render.

Reply:
ah good point!